agent/pkg/api/entry_streamer_socket_connector.go

```diff
@@ -22,7 +22,8 @@ func (e *DefaultEntryStreamerSocketConnector) SendEntry(socketId int, entry *tap
 	if params.EnableFullEntries {
 		message, _ = models.CreateFullEntryWebSocketMessage(entry)
 	} else {
-		extension := extensionsMap[entry.Protocol.Name]
+		protocol := protocolsMap[entry.ProtocolUniqueName]
```

Comment:
What if `protocolsMap` doesn't contain `entry.ProtocolUniqueName`?

Reply:
than i think it should crush, from the same reason if extensionsMap doesn't have the protocol name

Reply:
Let's check it and handle the error.